Neos.Flow/Classes/ResourceManagement/PersistentResource.php

```diff
@@ -285,6 +322,14 @@ public function setFileSize($fileSize)
      */
     public function getFileSize()
     {
+        if (!$this->fileSize && $this->source !== null) {
```

Comment:
This (and below lazy evaluations) is maybe not needed, but I'm unsure if the resource properties could be accessed before it being "imported" via the resource manager